.gitattributes

```
@@ -1 +1,3 @@
-tests/benchmarks/* linguist-vendored
\ No newline at end of file
+*.art linguist-language=Red
+*.art linguist-vendored
+tests/benchmarks/* linguist-vendored
```

```suggestion
tests/benchmarks/* linguist-vendored
tests/errors/* linguist-vendored
tests/unittests/* linguist-vendored
examples/rosetta/* linguist-vendored
```

While we are at it, would you mind adding these as well? These are - I think - all of the locations where `.res` files can be found. This way, we'll finally get rid of this "ReScript" (which I hadn't heard, until it popped up in the repo lol) thing as well. 😉